packages/core/src/common/commands-common.ts

```diff
@@ -8,6 +8,7 @@
 import { MAIN_MENU_BAR, MenuContribution, MenuModelRegistry } from './menu';
 import { CommandContribution, CommandRegistry } from './command';
 import { injectable } from "inversify";
+import { getMessage } from './messages';
```

Comment:
Could we rename `getMessage` to `localize`?

Reply:
Changes all of them and just updated my branch